### PR TITLE
fix(tabs): fix dropdown input heights, icon and hover states, Closes #1451

### LINKED ISSUE #1451: Tabs (Dropdown Issue) - experimental component (feedback from review)
When Tabs component reverts to Dropdown at Md breakpoint:

- [x] Make dropdown field Gray 20 on select (i.e. when menu is open) 
- [x] Change field height of Dropdown to 40px and change menu item heights to 40px as well!
- [x] Make sure hover states on open Dropdown menu will be #e5e5e5 (Gray 10 hover)
- [x] Make sure hover state covers the rules (see file and also, structured list component); i.e. you don't want to see hover state AND rules at the same time
- [x] Use chevron glyph (can ask @joshblack if you have trouble finding) for icon
- [x] Position chevron 16px from the right edge of the dropdown
 

[Drop_down_Specs_revised.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/43694f63-23e1-4192-aab8-d9bad950de3e)

### PR DESCRIPTION
**Pull request by jnm2377**
Closes #1451 

Resolves small design changes from last sprint demo feedback

#### Changelog

**Changed**
- Hover background color for menu items
- Selected item background color on open
- Icon: now chevron for experimental
- Input height: now `40px`

#### Testing / Reviewing
Ensure that selected item background color changes on open/close.
Ensure icon is arrow for standard component and chevron for experimental.
Ensure icon is transitioning correctly.

Staging link: http://carbon-dev-environment--tabs-x-wacky-crocodile.mybluemix.net/?nav=tabs

### CODE REVIEW

File: src/components/tabs/_tabs.scss [hunk 1]
```diff
@@ -340,7 +345,10 @@
       }
     }
     @include max-breakpoint(bp--sm--major) {
-      height: rem(34px);
+      height: rem(40px);
+      &:first-of-type {
+        margin-top: rem(1px);
```
Comment (asudoh) [inline src/components/tabs/_tabs.scss]: Just wondered if we want spacing scale like `$spacing-4xs`.

File: src/components/tabs/tabs.hbs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <nav data-tabs class="{{@root.prefix}}--tabs" role="navigation">
   <div class="{{@root.prefix}}--tabs-trigger" tabindex="0">
     <a href="javascript:void(0)" class="{{@root.prefix}}--tabs-trigger-text" tabindex="-1"></a>
-    <svg width="10" height="5" viewBox="0 0 10 5">
-      <path d="M0 0l5 4.998L10 0z" fill-rule="evenodd" />
+    <svg width="12" height="7" viewBox="0 0 12 7">
```
Comment (asudoh) [inline src/components/tabs/tabs.hbs]: Should this caret to chevron change be applied to stable, too?

**Review by asudoh — COMMENTED** [review]
LGTM basically, just one comment (spacing question can be disregarded) - Thanks @jnm2377!

File: src/components/tabs/_tabs.scss (hunk 1 above)
Comment (jnm2377) [reply src/components/tabs/_tabs.scss]: Good question. I hadn't seen a `$spacing-4xs` token but I can add it. Thanks @asudoh

File: src/components/tabs/tabs.hbs (hunk 2 above)
Comment (jnm2377) [reply src/components/tabs/tabs.hbs]: Good catch. I updated it to have an if statement for different icons 👍

**Review by aagonzales — APPROVED** [review]
Looks perfect! 💯 👍

**Review by joshblack — APPROVED** [review]
🎉

**Pull request merged**